docker/Dockerfile

```diff
@@ -1,9 +1,9 @@
 FROM debian:9.6-slim
 
-# Default google-fluentd at the unstable version.
+# Default google-fluentd at the stable version.
```

Comment:
Since this is a specific version, let's name it explicitly, rather than just "stable".

Reply:
Done.

Reply:
I was suggesting actually naming the version that corresponds to that REPO_SUFFIX, i.e., `1.6.2-1`.

Reply:
That info is automatically generated now: https://github.com/GoogleCloudPlatform/google-fluentd/blob/master/docker/Dockerfile#L25. We don't need to manually update it any more.

Reply:
But we do, because we'd have to update the repo suffix now. Once that happens, we might as well update the version as well (especially since the REPO_SUFFIX bump will likely happen automatically anyway).